libheif/codecs/uncompressed_box.cc

```diff
@@ -361,3 +361,91 @@ Error Box_uncC::write(StreamWriter& writer) const
 
   return Error::Ok;
 }
+
+
+Error Box_cmpC::parse(BitstreamRange& range)
+{
+  parse_full_box_header(range);
```

Comment:
We could add a version check like here: https://github.com/strukturag/libheif/commit/2de879aadb52c7d2f661c0067faa6b853330a9f1

Reply:
Fixed.